src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -23,7 +23,7 @@ export type ReactElement = {
   $$typeof: any,
   type: any,
   key: any,
-  ref: any,
+  ref: string | (elem: ?HTMLElement) => void,
```

Comment:
Expanding the types of `ReactElement` as I go along. Flow coverage tool in nuclide is super useful to see what flows sees in the code I flowify :)